api/src/main/java/org/pmiops/workbench/access/AccessTierServiceImpl.java

```diff
@@ -58,6 +59,21 @@ public AccessTierServiceImpl(
         .collect(Collectors.toList());
   }
 
+  /**
+   * Return all access tiers in the database which are visible to users, in alphabetical order by
+   * shortName. To return all tiers, use getAllTiers().
+   *
+   * @return the List of all DbAccessTiers in the database which are visible to users
+   */
+  @Override
+  public List<DbAccessTier> getAllTiersVisibleToUsers() {
```

Comment:
Just for my benefit, this will return ALL tiers from database (before filtration ), not just the one the user is eligible to

Reply:
there are 4 levels here, each getting smaller or staying the same
1. tiers which exist in the DB
2. tiers which exist in the DB and that users can see
3. tiers which the user is eligible for
4. tiers which the user has access to

This returns #2 - please LMK if there is a better name or wording that I could use here